baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -396,6 +396,19 @@ def get_snapshot_link(self):
             self.config["snapshot_url"] = snapshot_metadata["url"]
             self.config["snapshot_sha256"] = snapshot_metadata.get("sha256", None)
             self.config["snapshot_block_hash"] = snapshot_metadata["block_hash"]
+            print(
+                color(
+                    f"""
+Snapshot metadata:
+url: {snapshot_metadata["url"]}
+sha256: {snapshot_metadata["sha256"]}
+filesize: {snapshot_metadata["filesize"]}
+block height: {snapshot_metadata["block_height"]}
+block timestamp: {snapshot_metadata["block_timestamp"]}
```

Comment:
what do you mean by localized?

Reply:
I mean that it's printed in a way that uses the standard format of the running machine.

However, on second though, given that we only support English I think that it would be better to just use a format that is unambiguous, e.g. `Sep. 6th 2023`

Reply:
@krendelhoff2 another slight change of plan after discussing with Oxhead.

Please disregard the above and let's use a date-time format that it's localized to the system running the wizard.

Other standard linux programs and utilities do the same, so this would make them easier to visually compare.